docs/docs/setup/react-native-setup.md

```diff
@@ -10,9 +11,30 @@ In this tutorial, you will learn how to create a native Mopro module on both And
     <img src="/img/react-native-ios.png" alt="Second Image" width="250"/>
 </div>
 
-You have 2 options to get started with a mopro React Native project:
+You have 3 options to get started with a mopro React Native project:
+
+## Option 1: Use Mopro Cli
+
+The easiest way to set up your project is by using the mopro cli **create** command.<br/>This command helps you quickly add templates, similar to the next option, but with fewer manual steps.
+
+```sh
+mopro-example-app $ mopro create
+```
+
+Assuming you’ve successfully built the **ios** and **android** bindings: `MoproAndroidBindings` and `MoproIOSBindings` in your project folder, the mopro cli stored some parameters into the `Config.toml` file and reads them during the create command. It will also allow you to select the **react-native** template, as shown below:
+
+```
+? Create template ›
+  ios
+  android
+  web          - Require binding
+  flutter
+❯ react-native
+```
+
+## Option 2: Clone the Repository and Import the Bindings
 
-### Option 1: Clone the Repository and Import the Bindings
+This option is a more manual compared to [Option 1](docs/react-native-setup#option-1-use-mopro-cli). You can clone a pre-configured repository and manually import the generated bindings into your React Native project.
```

Comment:
You can just use hashtag to navigate within the page
```suggestion
This option is a more manual compared to [Option 1](#option-1-use-mopro-cli). You can clone a pre-configured repository and manually import the generated bindings into your React Native project.
```

Reply:
That’s why it keeps failing. I wasn’t aware that I was just running `yarn start instead` of `building` it. Thank you